Modules/HRV2/HRV2.h

```diff
@@ -0,0 +1,109 @@
+//
+// Created by Izabela Wilusz and Bartłomiej Ziółkowski on 08.12.2022.
+//
+
+#ifndef ECG_ANALYZER_HRV2_H
+#define ECG_ANALYZER_HRV2_H
+
+#include <complex>
+#include <iostream>
+#include <valarray>
+#include <vector>
+#include <cmath>
+#include <numeric>
+#include <cstdlib>
+#include <algorithm>
+#include <memory>
+using namespace std;
+
+struct HistogramData
+{
+    int bins;
+    vector<int> histogram;
+    vector<double> binBorders;
+    double interval;
+    int maxBinSize;
+    int maxBinNumber;
+};
+
+struct TinnParams{
+
+    vector<double> xVectorMT;
+    vector<double> yVectorMT;
+    vector<double> xVectorMN;
+    vector<double> yVectorMN;
+    vector<double> xVectorTN;
+    vector<double> yVectorTN;
+    double TinnLength;
+
+};
+
+struct PoincareParameters
+{
+    double sd1;
+    double sd2;
+};
+
+struct EllipseParameters
+{
+    vector<int> valuesSD1;
+    vector<int> valuesSD2;
+    vector<int> indicesSD1;
+    vector<int> indicesSD2;
+};
+
+class HRV2 {
```

Comment:
Wszystkie miejsca w których korzystacie z danych pochodzących z innych modułów powinny oczekiwać `std::shared_ptr` - pozwoli to nam na minimalizację operacji kopiowania dużych wektorów. 
Jeżeli wasz moduł zwraca informacje dla wizualizacji/GUI/innych modułów nie powinniście zwracać smart pointerów na poszczególne zmienne - wystarczą zwykłe wektory, czy podstawowe typy.
Końcowo zostało jeszcze dodanie w komentarzach opisu użycia waszego modułu - przykłady takich komentarzy znajdziecie w już zmergowanych modułąch - DataAccessor, ECG_Baseline itd.